work-scheduler-service/src/main/java/com/github/gaalgergo/workscheduler/service/employees/EmployeeServiceImpl.java

```diff
@@ -20,6 +21,9 @@
     @Autowired
     private ConversionService conversionService;
 
+    @Autowired
+    private EmployeeConverter employeeConverter;
```

Comment:
You don't need a separate `EmployeeConverter` instance. It will automatically be part of Spring's `ConversionService`.
Remember, the `conversionService` instance will call `EmployeeEntityConverter` in the background during the runtime of the `getAll()` method.